### PR TITLE
Ep 10415  restrict messages domain

### PR DESCRIPTION
**Pull request by drewniu**

### CODE REVIEW

File: src/bridge.js [hunk 1]
```diff
@@ -16,8 +16,15 @@ import { processEvent } from './events';
 import EnplugError from './errors/EnplugError';
 
 
+
 // todo finish reject timeout
 const RESPONSE_TIMEOUT = (60 * 1000);
+const VERSION = '0.3.8-dev14';
```
Comment (A-Gochnio) [inline src/bridge.js]: Consider importing package.json and using version property here.
Or don't and I'll do it in the refactor :)

File: src/bridge.js [hunk 2]
```diff
@@ -72,15 +79,27 @@ try {
 } catch (error) {
   // epBridge was not found. In such case, we assume that the application is iframed within
   // WebPlayer and communication has to proceed via posting and receiving messages between windows.
-  // TODO(michal): generalize hardcoded player.enplug.loc URL.
-  console.info('Initializing Web Development Player.');
+  console.info(`[Player SDK: ${VERSION}] Initializing Web Development Player.`);
+  const destinationMatch = window.location.href.match(/(https\:\/\/[a-z]*\.[a-z]*\.[a-z]{3})/);
```
Comment (A-Gochnio) [inline src/bridge.js]: This won't match "https://player.enplug.in" (last group requires 3 characters)

**Review by A-Gochnio — APPROVED** [review]

File: src/bridge.js (hunk 2 above)
Comment (drewniu) [reply src/bridge.js]: Done

File: src/bridge.js (hunk 1 above)
Comment (drewniu) [reply src/bridge.js]: I'll leave it as is for now

**Pull request merged**